cerbero/build/cookbook.py

```diff
@@ -112,6 +112,7 @@ def set_config(self, config):
         @type config: L{cerbero.config.Config}
         '''
         self._config = config
+        config.cookbook = self
```

Comment:
why this change on the cookbook is needed? I mean why config should have an instance of the cookbook?